standalone-metastore/pom.xml

```diff
@@ -79,8 +79,8 @@
     <dropwizard-metrics-hadoop-metrics2-reporter.version>0.1.2
     </dropwizard-metrics-hadoop-metrics2-reporter.version>
     <dropwizard.version>3.1.0</dropwizard.version>
-    <guava.version>19.0</guava.version>
-    <hadoop.version>3.1.0</hadoop.version>
+    <guava.version>27.0-jre</guava.version>
+    <hadoop.version>3.2.1</hadoop.version>
```

Comment:
I think we're targeting 3.3.1 here too, right?

Reply:
Wow, great catch.  Nooooooooooo!  Ugh.

I hope it doesn't break anything.